packages/stream_video/lib/src/call/call.dart

```diff
@@ -550,6 +554,20 @@ class Call {
     _stateManager.updateCallPreferences(preferences);
   }
 
+  void enableClientCapabilities(
+    List<SfuClientCapability> capabilities,
+  ) {
+    _logger.i(() => '[enableClientCapabilities] $capabilities');
+    capabilities.forEach(_sfuClientCapabilities.add);
+  }
+
+  void disableClientCapabilities(
+    List<SfuClientCapability> capabilities,
+  ) {
+    _logger.i(() => '[disableClientCapabilities] $capabilities');
+    capabilities.forEach(_sfuClientCapabilities.remove);
+  }
+
```

Comment:
I think this is public API right? If I understand correctly you can only do this before you join the call? We should document that.